### PR TITLE
chore: Improve ObjectHashAggregate fallback error message

### PR DESCRIPTION
**Pull request by andygrove**
## Which issue does this PR close?

Closes https://github.com/apache/datafusion-comet/issues/845

## Rationale for this change

Before this PR, we see errors such as:

```
ObjectHashAggregate [COMET: ObjectHashAggregate is not supported]
```

With this PR, we see information about which aggregate expression is not supported:

```
ObjectHashAggregate [COMET: unsupported Spark aggregate function: bloom_filter_agg]
```

### CODE REVIEW

**Comment by andygrove** [comment]
@parthchandra fyi, another small improvement in reporting unsupported expressions

**Review by kazuyukitanimura — APPROVED** [review]

**Review by eejbyfeldt — APPROVED** [review]

File: spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala [hunk 1]
```diff
@@ -2663,16 +2663,16 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
...
-            _,
-            groupingExpressions,
-            aggregateExpressions,
-            aggregateAttributes,
-            _,
-            resultExpressions,
-            child) if isCometOperatorEnabled(op.conf, CometConf.OPERATOR_AGGREGATE) =>
+      case aggregate: BaseAggregateExec
+          if (aggregate.isInstanceOf[HashAggregateExec] ||
+            aggregate.isInstanceOf[ObjectHashAggregateExec]) &&
```
Comment (viirya) [inline spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala:2668]: So for `ObjectHashAggregateExec`, the error message will be something like unsupported aggregate expression?

**Review by viirya — APPROVED** [review]
Oh, I saw it in the PR description.

File: spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala (hunk 1 above)
Comment (viirya) [reply spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala:2668]: Oh, I saw it in the PR description.

**Pull request merged**